src/main/java/frc/robot/commands/ArmJointControlCommand.java

```diff
@@ -0,0 +1,71 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotMap;
+import frc.robot.subsystems.ArmJointSystem;
+
+import java.util.PrimitiveIterator;
+
+public class ArmJointControlCommand extends Command {
+    private final ArmJointSystem sub;
+    private double targetPosition;
+    private boolean isInTarget;
+    private boolean hasNewPosition;
+    private TrapezoidProfile motionProfile;
+    private TrapezoidProfile.State motionProfileGoal;
+    private TrapezoidProfile.State motionProfileSetPoint;
+
+    public ArmJointControlCommand(ArmJointSystem sub) {
+        this.sub = sub;
+
+        addRequirements(sub);
+    }
+
+    @Override
+    public void initialize() {
+        isInTarget = false;
+        hasNewPosition = true;
+        targetPosition = 0;
```

Comment:
Add support for "not holding" mode and use it initially. In this mode the arm is not forcibly held by PID, and the motor is idle. Useful as default and when we don't need to keep the arm raised.

Add support in the command to enter this "not holding" mode via a method, like with asking for a specific position.

Automatically transition to this mode when the arm is next to the floor (there are two floor position as there are two sides, one would be around 0 and the other likely 180).